contrib/scripts/createmodule.py

```diff
@@ -36,10 +36,21 @@
     parser.print_usage()
     exit(1)
 
+# Determine if running environment is based on cmd.exe or not
+def iscmdshell():
+    import platform
+    return True if platform.system().lower() == 'windows' else False
```

Comment:
Seems result string does not vary across Python version or platform, so the `lower()` transformation is not needed. From my understanding just checking it equals to `'Windows'` should be ok